From cc488a228ca4eee7c62ddb08e49cdee46365a0e6 Mon Sep 17 00:00:00 2001
From: Ren Jianing <jianing.ren@rock-chips.com>
Date: Wed, 11 Nov 2020 20:32:14 +0800
Subject: [PATCH] ohci-platform: add the max clock number to 4

Rockchip SoCs such as RV1126 and RK356x requires
4 clocks to be enabled for OHCI.

Signed-off-by: Ren Jianing <jianing.ren@rock-chips.com>
Change-Id: Ia5202ca7223d95a4b39b1581f740e03ca3f54224
---
 drivers/usb/host/ohci-platform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/ohci-platform.c b/drivers/usb/host/ohci-platform.c
index c70ec9daf9de..e11eb9c81876 100644
--- a/drivers/usb/host/ohci-platform.c
+++ b/drivers/usb/host/ohci-platform.c
@@ -32,7 +32,7 @@
 #include "ohci.h"
 
 #define DRIVER_DESC "OHCI generic platform driver"
-#define OHCI_MAX_CLKS 3
+#define OHCI_MAX_CLKS 4
 #define hcd_to_ohci_priv(h) ((struct ohci_platform_priv *)hcd_to_ohci(h)->priv)
 
 struct ohci_platform_priv {
-- 
2.35.3

